techmatters/terraso-mobile-client
# feat: offline sites list

## Description

Make offline version of sites list compliant with design.

### Checklist

### Related Issues
#2369

### Verification steps

Open the sites screen in an empty and non-empty account. Verify that the alert appears with the correct text for both in offline mode.
